vllm/model_executor/models/aria.py

```diff
@@ -494,9 +558,12 @@ def input_processor(ctx, llm_inputs):
             repeat_count=num_crops,
         )
 
-    repeat_count = [hf_config.image_size2tokens[max_image_size]
-                    ] * sum(num_crops).item()
-    new_prompt, new_token_ids, _ = repeat_and_pad_placeholder_tokens(
+    image_size2tokens = {
+        int(math.sqrt(k) * hf_config.vision_config.patch_size): v
+        for k, v in hf_config.projector_patch_to_query_dict.items()
+    }
```

Comment:
Seems that this is a fixed value, perhaps we can move it to `AriaMoELMConfig` initialization in `vllm/transformers_utils/configs/aria.py` to avoid repeat calculation?

Reply:
Yep I can do that

Reply:
I realized we actually don't need this calculation `int(math.sqrt(k) * hf_config.vision_config.patch_size` at all since we only care about the values here, so I will just simplify this.